tests/framework/installer/ceph_manifests_previous.go

```diff
@@ -37,8 +42,16 @@ func (m *CephManifestsPreviousVersion) Settings() *TestCephSettings {
 	return m.settings
 }
 
-func (m *CephManifestsPreviousVersion) GetCRDs(k8shelper *utils.K8sHelper) string {
-	return m.settings.readManifestFromGitHub("crds.yaml")
+func (m *CephManifestsPreviousVersion) GetCRDs(t *testing.T, k8shelper *utils.K8sHelper) string {
```

Comment:
The `CephManifestsPreviousVersion` type is only used for the upgrade tests. During the upgrade tests we always want to use the crds.yaml since the previous version (1.12) does not have the pre-1.25 crds. So the upgrade test should pass if you revert the changes to this file.